docs/sql-ref-syntax-ddl-create-table-hiveformat.md

```diff
@@ -96,21 +97,37 @@ CREATE [ EXTERNAL ] TABLE [ IF NOT EXISTS ] table_identifier
 ### Examples
 {% highlight sql %}
 
---Using Comment and loading data from another table into the created table
-CREATE TABLE StudentInfo
```

Comment:
the previous example was wrong. By default `CREATE TABLE` without USING created data source table in 3.0.

Reply:
I think this is an important change for users, so how about describing the difference in 2.4/3.0 here, too?

Reply:
I think migration guide should be responsible for it ..

Reply:
I remember we already add a note for this in migration guide.

Reply:
yes we have. Search `CREATE TABLE` in https://github.com/apache/spark/blob/master/docs/sql-migration-guide.md

Reply:
yea, ok. thanks